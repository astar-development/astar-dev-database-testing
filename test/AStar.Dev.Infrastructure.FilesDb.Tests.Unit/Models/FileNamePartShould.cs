using AStar.Dev.Infrastructure.FilesDb.Models;
using AStar.Dev.Utilities;
using Shouldly;

namespace AStar.Dev.Infrastructure.FilesDb.Tests.Unit.Models;

public class FileNamePartShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new FileNamePart { FileClassifications = new List<FileClassification> { new()  { Id = 1 } } }
           .ToJson()
           .ShouldMatchApproved();
}
