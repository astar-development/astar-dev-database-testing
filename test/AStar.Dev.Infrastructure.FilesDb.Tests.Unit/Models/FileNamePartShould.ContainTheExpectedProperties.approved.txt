{
  "id": 0,
  "text": "",
  "fileClassifications": [
    {
      "id": 1,
      "name": "",
      "fileDetails": [],
      "celebrity": false,
      "fileNameParts": []
    }
  ]
}